samples/gke/README.adoc

```diff
@@ -0,0 +1,96 @@
+= GKE
+
+[horizontal]
+*Language*:: JavaScript
+*Function*:: https://github.com/markfisher/sk8s/blob/master/samples/gke/slack-command/slack_command.js[slack_command.js]
+*Protocol*:: HTTP
+*Input*:: slack
+*Output*:: none (replies, implicitly)
+
+=== TL;DR
+
+If you have push access to the `sk8s` org on dockerhub, you can probably run the `test.sh` script in this
```

Comment:
I think we should assume that readers don't have this access

Reply:
what does that mean, concretely? Would you say something like
> If you have push access, ..., but you probably don't.
? Would you drop any mention of the test script?

Reply:
I would leave the test script out of this README unless it is easily runnable by all

Reply:
👍